Source/WebCore/rendering/MotionPath.cpp

```diff
@@ -81,8 +72,7 @@ std::optional<MotionPathData> MotionPath::motionPathDataForRenderer(const Render
             auto usedStartingPosition = startingPosition.x().isAuto() ? startingPositionForOffsetPosition(offsetPosition, pathReferenceBoxRect, *container) : floatPointForLengthPoint(startingPosition, pathReferenceBoxRect.size());
             data.usedStartingPosition = usedStartingPosition;
             data.offsetFromContainingBlock = offsetFromContainer(renderer, *container, pathReferenceBoxRect);
-            data.containingBlockBoundingRect = pathReferenceBoxRect;
-
+            data.containingBlockBoundingRect = FloatRoundedRect(pathReferenceBoxRect);
```

Comment:
Can we avoid duplicating the code that computes the pathReferenceBoxRect ?

Reply:
Refactored.